api/src/dtos/cards.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { PriceHistoryData } from '@/interfaces/priceHistory.interface';
+import { IsEmail, IsString, IsNotEmpty, MinLength, MaxLength } from 'class-validator';
+
+// Will add a new rating to the list of ratings adjusting the average
+export class CreateRatingDto {
+  public rating: number;
```

Comment:
Probably want to add that it's required - I think there's one from the class validator